terraform/ecs_task.tf

```
@@ -26,7 +26,8 @@ locals {
     { name = "DB_SKIP_MIGRATION", value = "true" },
     { name = "USE_SSL_FOR_DB", value = "true" },
     { name = "LOG_LEVEL", value = var.log_level },
-    { name = "SQS_EHR_OUT_INCOMING_QUEUE_URL", value = aws_sqs_queue.service_incoming.id }
+    { name = "SQS_EHR_OUT_INCOMING_QUEUE_URL", value = aws_sqs_queue.service_incoming.id },
+    { name = "EHR_REPO_RATE_LIMIT_TIMEOUT_MILLISECONDS", value = "1000" }
```

```suggestion
    { name = "FRAGMENT_TRANSFER_RATE_LIMIT_TIMEOUT_MILLISECONDS", value = "1000" }
```
I believe it's more suitable to called it "fragment transfer" rather than "ehr-repo", as the timeout is occuring within the transferring out process rather than within ehr-repo.